qt/scientific_interfaces/Indirect/test/FqFitModelTest.h

```diff
@@ -130,15 +98,6 @@ class FqFitModelTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(m_model->getWidths(TableDatasetIndex{0})[1], "f1.FWHM");
   }
 
-  void test_that_getEISF_will_return_an_empty_vector_if_there_are_no_EISFs() {
```

Comment:
This should still be left tyhere. and can you write an equivilant test for getWidths